tests/pipelines/test_pipelines_question_answering.py

```diff
@@ -34,11 +34,20 @@
 from .test_pipelines_common import ANY
 
 
+# These 2 model types require different inputs than those of the usual text models.
+_TO_SKIP = {"LayoutLMv2Config", "LayoutLMv3Config"}
+
+
 @is_pipeline_test
 class QAPipelineTests(unittest.TestCase):
     model_mapping = MODEL_FOR_QUESTION_ANSWERING_MAPPING
     tf_model_mapping = TF_MODEL_FOR_QUESTION_ANSWERING_MAPPING
 
+    if model_mapping is not None:
+        model_mapping = {config: model for config, model in model_mapping.items() if config.__name__ in _TO_SKIP}
+    if tf_model_mapping is not None:
+        tf_model_mapping = {config: model for config, model in tf_model_mapping.items() if config.__name__ in _TO_SKIP}
```

Comment:
This is to avoid the changes of `pipeline_model_mapping` in this PR being reverted next time we update the repository with the script in `add_pipeline_model_mapping_to_test.py`

Reply:
This also makes things more explicit: those models are not for this pipeline (test)